src/Effects.js

```diff
@@ -1254,6 +1255,37 @@ module.exports = class Effects {
       }
     };
 
+    this.exploding_stars = {
+      stars: [],
+      initStars: function () {
+        for (let i = 0; i < 100; i++) {
+          let theta = p5.random(0, p5.TWO_PI);
+          let velocity = p5.random(4,12);
+          this.stars.push({
+            color: randomColor(255, 255, 100),
```

Comment:
Can we use the color palette to determine the star color? I think we can use 'randomColorFromPalette'.

Reply:
The foreground block doesn't have a palette, and it seemed like none of the other foreground effects use the palette. I could change it to match the selected palette if there is one, but if there's no palette selected fall-back to random? That seems a little weird though because then the foreground block depends on the value from the background block? What do you think?

Reply:
You're totally right - I just forgot this was a foreground. Ignore me!